src/main/java/forestry/apiculture/EventHandlerApiculture.java

```diff
@@ -34,19 +34,43 @@ public void addFrameTooltip(ItemTooltipEvent event) {
 
                     float territory = modifier.getTerritoryModifier(null, 1.0F);
-                    tooltip.add(StringUtil.localize("frame.tooltip.territory") + getColorFormatted(territory));
-
+                    if (territory != 1.0F) {
+                        tooltip.add(
+                                StringUtil.localize("frame.tooltip.territory")
+                                        + getModifierFormatted(territory, false, false));
+                    }
                     float mutation = modifier.getMutationModifier(null, null, 1.0F);
-                    tooltip.add(StringUtil.localize("frame.tooltip.mutationRate") + getColorFormatted(mutation));
-
+                    if (mutation != 1.0F) {
+                        tooltip.add(
+                                StringUtil.localize("frame.tooltip.mutationRate")
+                                        + getModifierFormatted(mutation, false, false));
+                    }
                     float lifespan = modifier.getLifespanModifier(null, null, 1.0F);
-                    tooltip.add(StringUtil.localize("frame.tooltip.lifespan") + getColorFormatted(lifespan));
-
-                    float production = modifier.getProductionModifier(null, 1.0F);
-                    tooltip.add(StringUtil.localize("frame.tooltip.production") + getColorFormatted(production, true));
-
+                    if (lifespan != 1.0F) {
+                        // lower lifespan is better
+                        tooltip.add(
+                                StringUtil.localize("frame.tooltip.lifespan")
+                                        + getModifierFormatted(lifespan, false, true));
+                    }
+                    float production = modifier.getProductionModifier(null, 0.0F);
+                    if (production != 0.0F) {
+                        tooltip.add(
+                                StringUtil.localize("frame.tooltip.production")
+                                        + getModifierFormatted(production, true, false));
+                    }
+                    float flowering = modifier.getProductionModifier(null, 1.0F);
+                    if (flowering != 1.0F) {
+                        tooltip.add(
+                                StringUtil.localize("frame.tooltip.flowering")
+                                        + getModifierFormatted(flowering, false, false));
+                    }
                     float decay = modifier.getGeneticDecay(null, 1.0F);
-                    tooltip.add(StringUtil.localize("frame.tooltip.geneticDecay") + getColorFormatted(decay));
+                    if (decay != 1.0F) {
+                        // lower genetic decay is better
+                        tooltip.add(
+                                StringUtil.localize("frame.tooltip.geneticDecay")
+                                        + getModifierFormatted(decay, false, true));
+                    }
```

Comment:
Why are you hiding modifiers that does not change anything? (x1, +0) I think it looks worse now that they are hidden. Please revert.